js/index.js

```diff
@@ -42,28 +43,117 @@ function renderPepperoni() {
   });
 }
 
-function renderMushrooms() {
-  // Iteration 1: set the visibility of `<section class="mushroom">`
-}
+
 
 function renderGreenPeppers() {
-  // Iteration 1: set the visibility of `<section class="green-pepper">`
-}
+  
+    document.querySelectorAll('.green-pepper').forEach(greenPep => {
+      if (state.greenPeppers) {
+        greenPep.style.visibility = 'visible';
+      } else {
+        greenPep.style.visibility = 'hidden';
+      }
+    });
+  }
+  
+
+  function renderMushrooms() {
+  
+    document.querySelectorAll('.mushroom').forEach(mush=> {
+      if (state.mushrooms) {
+        mush.style.visibility = 'visible';
+      } else {
+        mush.style.visibility = 'hidden';
+      }
+    });
+  }
+  
+
 
 function renderWhiteSauce() {
   // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
+
+  document.querySelectorAll('.sauce').forEach(sauce=> {
+    if (state.whiteSauce) {
+      sauce.classList.add('sauce-white');
+    } else {
+      sauce.classList.remove('sauce-white');
+    }
+  });
 }
 
 function renderGlutenFreeCrust() {
   // Iteration 2: add/remove the class "crust-gluten-free" of `<section class="crust">`
+
+    document.querySelectorAll('.crust').forEach(crust=> {
+      if (state.glutenFreeCrust) {
+        crust.classList.add('crust-gluten-free');
+      } else {
+        crust.classList.remove('crust-gluten-free');
+      }
+    });
+  
 }
+  
 
 function renderButtons() {
   // Iteration 3: add/remove the class "active" of each `<button class="btn">`
+
+  document.querySelectorAll('.btn-pepperoni').forEach(button => {
+    if (state.pepperoni) {
+      button.classList.add('active');
+    } else {
+      button.classList.remove('active');
+    }
+  });
+
+
+document.querySelectorAll('.btn-mushrooms').forEach(button => {
+  if (state.mushrooms) {
+    button.classList.add('active');
+  } else {
+    button.classList.remove('active');
+  }
+});
+
+document.querySelectorAll('.btn-green-peppers').forEach(button => {
+  if (state.greenPeppers) {
+    button.classList.add('active');
+  } else {
+    button.classList.remove('active');
+  }
+});
+
+document.querySelectorAll('.btn-sauce').forEach(button => {
+  if (state.glutenFreeCrust) {
+    button.classList.add('active');
+  } else {
+    button.classList.remove('active');
+  }
+});
+
+document.querySelectorAll('.btn-crust').forEach(button => {
+  if (state.whiteSauce) {
+    button.classList.add('active');
+  } else {
+    button.classList.remove('active');
+  }
+});
```

Comment:
here you just change the name of the propriety of your state
```glutenFreeCrust < = > whiteSauce```
so this is why when you click in one button change the class of the other one